app.py

```diff
@@ -17,7 +17,11 @@ def get_high_returns():
         request.json["lag_return"],
         request.json["daily_volume"],
     )
-    df = momentum_scanner_intraday.get_high_returns(dex, lag_return, daily_volume)
+    if 'marketcap' in request.json().keys() and 'vol_30' in request.json().keys() :
```

Comment:
Why are these optional? Why do we ignore one if only one is included. It seems possible this should be a separate API, but either way it should be an error to include only e.g. `dex, lag_return, daily_volume, marketcap`.

Is `market_cap` one word or two? If two it should be separated by a `_`.

Reply:
Changed to market_cap

Reason for making those inputs optional is to keep the current process intact. If the request doesn't have them it will use default values.

Reply:
To be clear, this says
```
                                     VVV
'marketcap' in request.json().keys() and 'vol_30' in request.json().keys()
```

Which means that if I include the key `"market_cap"` BUT NOT the key `"vol_30"` then it will silently ignore my key. If the API requires both or neither it should be an error to include only one.

Reply:
Let's just go without the if's and assume request contains all keys?